### ISSUE TITLE
Xrm.Attribute<any> instead of Xrm.OptionSetAttribute<boolean> generated for custom form

### ISSUE THREAD

**Issue by mariusstaub** [delegateas/XrmDefinitelyTyped#214]
**Description**
I have added a custom account form and when I generate the type information with `XrmDefinitelyTyped` it generates `Xrm.Attribute<any>` instead of `Xrm.OptionSetAttribute<boolean>` for all "boolean" fields (e.g. `donotemail`) on the custom form. The type information for the default form is as expected.

For the default form I get this line in the .d.ts file:
```
interface Attributes extends Xrm.AttributeCollectionBase {
  [...]
  get(name: "donotemail"): Xrm.OptionSetAttribute<boolean>;
  [...]
}
```

For the custom form I get this line in the .d.ts file:
```
interface Attributes extends Xrm.AttributeCollectionBase {
  [...]
  get(name: "donotemail"): Xrm.Attribute<any>;
  [...]
}
```
I would expect that the type of `donotemail` is always `Xrm.OptionSetAttribute<boolean>`.

**Environment**
- CRM/D365/CDS version: 9.1
- Tool version: v.5.4.7

**Question**
- What can I do to debug this issue? Can I turn on log files which could give me a hint?

Thank you for the work on this very helpful tool!

**Comment by misoeli** [comment]
Just checked in 5.5.1, Not able to reproduce. 
Could you retry? If the issue persists, could you elaborate on process of creating the form: Which form did you duplicate? (For Account, there's too many OotB forms for me to be able to identify one specific as "default")

**Comment by misoeli** [comment]
I would expect this was the same issue as #232, that is, the field was shown in a custom control. That issue is fixed in 5.6.0.

**Comment by mariusstaub** [comment]
Thanks a lot! The issue is indeed fixed in 5.6.0
